pkg/services/dashboards/database/database_test.go

```diff
@@ -323,7 +323,8 @@ func TestIntegrationDashboardDataAccess(t *testing.T) {
 		require.Equal(t, len(hits), 1)
 		hit := hits[0]
 		require.Equal(t, hit.Type, model.DashHitFolder)
-		require.Equal(t, hit.URL, fmt.Sprintf("/dashboards/f/%s/%s", savedFolder.UID, savedFolder.Slug))
+		require.Equal(t, hit.UID, savedFolder.UID)
+		require.Equal(t, hit.FolderUID, "")
```

Comment:
It would be great to see tests for the alternate search query, ones comparable to the tests we have for search v1/v2. This seems particularly relevant since we are expecting a potential mismatch of results between search v1 and alt search.